index.html

```diff
@@ -25,12 +27,185 @@ <h2 class="padding">I’m Youssef Glad to see you!</h2>
+                <img src="./assets/Counter.svg" alt="" />
+                <span>Back End Dev</span>
+                <img src="./assets/Counter.svg" alt="" />
+                <span>2015</span>
+              </div>
+            </div>
+            <p class="card-descrip card-item-mg">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="card-tags card-item-mg">
+              <span class="card-tag">html</span>
+              <span class="card-tag">css</span>
+              <span class="card-tag">javascript</span>
+            </div>
+            <div class="card-action">
+              <button class="card-btn">See Project</button>
+            </div>
+          </div>
+        </div>
+        <!-- Third Card -->
+        <div class="card">
+          <img class="card-img" src="./assets/Snapshoot3.png" alt="" />
+          <div class="card-details">
+            <div class="card-item-mg">
+              <span class="card-title">Tonic</span>
+              <div class="card-info">
+                <span class="info-active">CANOPY</span>
+                <img src="./assets/Counter.svg" alt="" />
+                <span>Back End Dev</span>
+                <img src="./assets/Counter.svg" alt="" />
+                <span>2015</span>
+              </div>
+            </div>
+            <p class="card-descrip card-item-mg">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="card-tags card-item-mg">
+              <span class="card-tag">html</span>
+              <span class="card-tag">css</span>
+              <span class="card-tag">javascript</span>
+            </div>
+            <div class="card-action">
+              <button class="card-btn">See Project</button>
+            </div>
+          </div>
+        </div>
+        <!-- Fourth Card -->
+        <div class="card">
+          <img class="card-img" src="./assets/Snapshoot4.png" alt="" />
+          <div class="card-details">
+            <div class="card-item-mg">
+              <span class="card-title">Multi-Post Stories</span>
+              <div class="card-info">
+                <span class="info-active">CANOPY</span>
+                <img src="./assets/Counter.svg" alt="" />
+                <span>Back End Dev</span>
+                <img src="./assets/Counter.svg" alt="" />
+                <span>2015</span>
+              </div>
+            </div>
+            <p class="card-descrip card-item-mg">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="card-tags card-item-mg">
+              <span class="card-tag">html</span>
+              <span class="card-tag">css</span>
+              <span class="card-tag">javascript</span>
+            </div>
```

Comment:
- [ ] Please make sure you use the `ul` and `li` tags to list the tagged languages in your project. You can read more about it [here](https://www.w3schools.com/html/html_lists.asp) :pray:

Reply:
fixed